frontend-ssr/components/lessons/LessonComments/LessonComments.module.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../../styles/variables';
```

Comment:
I think we can get rid of this, right? 

Or, if you to add a border for testing/development purposes - add it inline so we don't need to add another file, just for that. And we'll delete it on the final integration.